src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -215,9 +215,9 @@ def _get_filepath_in_current_git_repo(filename):
     CWD = 0
     for walk in os.walk(_gitroot()):
         for file_name in walk[FILE_NAMES]:
-            if file_name == filename:
+            if file_name.lower(file_name) == file_name.lower(filename):
```

Comment:
This with throw - ```.lower()``` does not take any parameters. Make sure you run the code locally before submitting.